public/javascript/chat.js

```diff
@@ -13,55 +20,59 @@ function getTimeString(time) {
 }
 
 function escapeMessage(message) {
-	// Escape "<"
-	let lt_re = new RegExp("<", "g");
-	message = message.replace(lt_re, "&lt");
+  // Escape "<"
+  let lt_re = new RegExp('<', 'g');
+  message = message.replace(lt_re, '&lt');
 
-	// Escape ">"
-	let gt_re = new RegExp(">", "g");
-	message = message.replace(gt_re, "&gt");
+  // Escape ">"
+  let gt_re = new RegExp('>', 'g');
+  message = message.replace(gt_re, '&gt');
 
-	return message
+  return message;
 }
 
 function chatElements(currentMessage) {
-  currentText = currentMessage ? currentMessage : "";
-  return '<textarea id="inputBox" type="text" autocomplete="off" placeholder="Type a message...">'
-       + currentText + '</textarea>'
-       + '<div id="sendButton"><div id="sendButtonText">Send</div></div>';
+  let currentText = currentMessage ? currentMessage : '';
+  return(`
+    <textarea id='inputBox' type='text' autocomplete='off' placeholder="Type a message...">
+      ${currentText}
```

Comment:
Template literals (` quotes) add spaces and newlines when the code has those. So if currentText is the empty string, this will turn it into a 2 newlines with some miscellaneous spaces. ${currentText} needs to be right between the >< brackets